frontend/src/widgets/PercentageActivityReportByRole.js

```diff
@@ -24,7 +24,7 @@ const DEFAULT_SORT_CONFIG = {
   activePage: 1,
 };
 
-export default function PercentageActivityReportByRole({ data }) {
+export default function PercentageActivityReportByRole({ data, loading }) {
```

Comment:
I left the loading attribute off these graphs on purpose; we don't really want all those announcements/mini "loading" spinners since the whole page is wrapped in a loader. There's an outstanding accessibility ticket to fix it. everywhere.

Reply:
Ok I removed loaders from all of the individual widgets and just added one loader on the dashboard itself.

Reply:
This has been cleaned up.